### PR TITLE
feat: Add graphql-jit

### LINKED ISSUE #304: add a benchmark for GraphQL JIT
Add benchmarks for [GraphQL JIT](https://github.com/zalando-incubator/graphql-jit)

Add all the existing benchmarks

### PR DESCRIPTION
**Pull request by Dhanus3133**
Fixes #304

/claim #304

### CODE REVIEW

File: graphql/graphql_jit/server.js [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+    greet: String
+    posts: [Post]
+    users: [User]
+    post(id: Int!): Post
+    user(id: Int!): User
+  }
+`);
+
+const fetchData = async (url) => {
+  const fetch = (await import('node-fetch')).default;
```
Comment (tusharmath) [inline graphql/graphql_jit/server.js:34]: We should create a client once and setup its proxy settings to point to nginx.

**Comment by tusharmath** [comment]
Moving to draft to reduce noise and improve CI efficiency. Once you are ready just mark it as "ready to review". Feel free to give a shoutout on the #contributors channel on Discord if you want immediate attention.

File: graphql/graphql_jit/server.js (hunk 1 above)
Comment (Dhanus3133) [reply graphql/graphql_jit/server.js]: Updated.

File: graphql/graphql_jit/server.js [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+app.use('/graphql', async (req, res) => {
+  const query = req.body.query || req.query.query;
+  if (!query) {
+    res.status(400).send('Query not provided');
+    return;
+  }
+
+  try {
+    const document = parse(query);
+    const compiledQuery = compileQuery(schema, document);
```
Comment (tusharmath) [inline graphql/graphql_jit/server.js:76]: Should compile the query only once.

**Review by tusharmath — APPROVED** [review]

**Pull request merged**